### ISSUE TITLE
Support 3rd Party data in Person

### ISSUE THREAD

**Issue by daonb** [hasadna/Open-Knesset#315]
We are getting info from The Marker and 100 days of transparency and since it's *soft* data it is important to store the source of the info and the creation & update date.  I'm thinking of a new model that has a one-to-many pointer to person, key name, value, source and the dates.
It's important to make the new data available in the admin and in the API

**Comment by morchickit** [comment]
Is this even still relevant? Which 3rd party data are we getting today? @OriHoch

**Comment by OriHoch** [comment]
yes, it's relevant, tomer avital has a lot of data which we could incorporate

**Comment by daonb** [comment]
I have the data too, or rather osubs has it. here's a sample -
https://osubs.org/#/committee/4/#candidate-3700

On 28 December 2015 at 22:06, Ori Hoch <notifications@github.com> wrote:

> yes, it's relevant, tomer avital has a lot of data which we could
> incorporate
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/Open-Knesset/issues/315#issuecomment-167643694>
> .
>



-- 



http://tuzig.com - Building Fires in The Cloud
